Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -220,7 +220,7 @@ public NodejsProjectNode(NodejsProjectPackage package)
 
         public bool IsTypeScriptProject {
             get {
-                return string.Equals(GetProjectProperty(NodejsConstants.EnableTypeScript), "true", StringComparison.OrdinalIgnoreCase);
+                return IsProjectOpened && string.Equals(GetProjectProperty(NodejsConstants.EnableTypeScript), "true", StringComparison.OrdinalIgnoreCase);
```

Comment:
Hmm, I think we're still susceptible to race conditions here... why is GetProjectProperty hanging?